ocfweb/static/scss/pages/_tv.scss

```diff
@@ -12,6 +12,7 @@ $base-font-size: 8vw;
         .hours-container {
             font-size: $base-font-size + 12;
             line-height: 1em;
+            text-wrap: none;
```

Comment:
yeah, it's not even an implemented one that's supported on any browser. For some reason when I tried it it "worked" but that was because I was manually editing things in the Chrome Inspector. white-space: nowrap didn't work either because it just truncates any overflow. My current solution is hacky but it works - it just gives it a smaller font size overall if the hours would wrap. Apparently you can't do automatic resizing based on amount of text in CSS.